docs/guide/navigation/navigation.md

```diff
@@ -32,89 +25,23 @@ You can set up a different Location depth for content from the content model and
 
 ## Fetching the data
 
-Navigation uses the built-in search to fetch the content and a custom search service to fetch to catalog data directly from Solr.
-Every data provider must ensure that the data is indexed in the correct format.
-The following data must be indexed:
-
-``` 
-document_type_id: [content/econtent]
-type_id: [class id]
-main_location_visible_b: [true/false]
-section_id: [section id]
-main_location_depth_i: [depth]
-main_location_path_id: [path]
-main_location_priority_i: [priority]
-```
-
-Additionally, every data provider can extend the search query before the catalog data is fetched from Solr.
-See the [Cookbook](modifying_the_search_query.md) for more information.
-
-### Fields used for navigation
-
-``` yaml
-siso_core.default.navigation.catalog:
-    #the class id has to be specified here
-    types: ['38']
-    sections: [1, 2]
-    enable_priority_zero: true
-    label_fields: ['ses_category_ses_name_value_s','name_s']
-    additional_fields: ['ses_category_ses_code_value_s', 'ses_category_ses_name_value_s' ]
-```
-
-You can define the name used for navigation in configuration. The `label_fields` parameter contains a list of attribute names (Solr names) which are used as the name in the menu. The first available attribute is used. 
-
-!!! caution
-
-    The standard attribute `name_s` does not always contain the correct translation. 
-    When `name_s` is used in `label_fields`, navigation may not be translated.
-    To resolve this, configure the ID of the attribute directly e.g. `ses_category_ses_name_value_s`.
+Navigation uses the built-in search to fetch the content, and a custom search service to fetch catalog data directly from Solr.
 
-### Sorting
+### Modifying the search query
 
-The navigation elements are sorted by Location priority and you can enable or disable fetching of nodes where the priority is 0.
-The active node is highlighted in the navigation.
+You can extend the search query before navigation data is fetched
+using two events that are thrown before the search query is submitted:
```

Comment:
Is it "the data is fetched using two events" or "you can extend the query using"? If the latter, it would be best to split the sentence: "You can extend the search query before navigation data is fetched. You do this by using two events that are thrown before the search query is submitted:"